introduction.md

```diff
@@ -1,2 +1,5 @@
 Hello, my name is Mr C.Intestinali. Happy to be here.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?